src/client/actions/InteractionCreate.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Action = require('./Action');
+const { Events, InteractionTypes, MessageComponentTypes } = require('../../util/Constants');
+const Structures = require('../../util/Structures');
+
+class InteractionCreateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    this.getChannel(data);
```

Comment:
Does this...cache the channel as partial? Can we at least add a comment so whoever touches this file knows that this line isn't here just because? 🙏